templates/ts/spfx-tab/appPackage/manifest.json.tpl

```diff
@@ -30,15 +30,7 @@
             "contentUrl": "https://{teamSiteDomain}/_layouts/15/TeamsLogon.aspx?SPFX=true&dest=/_layouts/15/teamshostedapp.aspx%3Fteams%26personal%26componentId={{componentId}}%26forceLocale={locale}",
             "websiteUrl": "https://products.office.com/en-us/sharepoint/collaboration",
             "scopes": [
-                "personal"
-            ]
-        }
-    ],
-    "configurableTabs": [
-        {
-            "configurationUrl": "https://{teamSiteDomain}{teamSitePath}/_layouts/15/TeamsLogon.aspx?SPFX=true&dest={teamSitePath}/_layouts/15/teamshostedapp.aspx%3FopenPropertyPane=true%26teams%26componentId={{componentId}}%26forceLocale={locale}",
-            "canUpdateConfiguration": true,
-            "scopes": [
+                "personal",
                 "team"
```

Comment:
no group chat?

Reply:
Yes, group chat was not supported before, so this change will remain the same.